Tasks/VsTest/vstest.ts

```diff
@@ -456,22 +704,25 @@ function resetRunInParallel() {
     runInParallel = false;
 }
 
+function compareNumbers(a,b) {
+    return a - b;
```

Comment:
Why not use inbuilt sort functions. what are you trying to sort. Array or numbers or something else?

---
In reply to: [68178941](https://github.com/Microsoft/vsts-tasks/pull/1932#discussion_r68178941) [](ancestors = 68178941)